specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+      "type": "object",
+      "properties": {
+        "nodes": {
+          "type": "array",
+          "description": "Information about nodes in the cluster (corresponds to what is returned from nodetool info).",
+          "items": {
+            "type": "object",
+            "properties": {
+              "datacenter": {
+                "type": "string",
+                "description": "The Cassandra data center this node resides in."
+              },
+              "status": {
+                "type": "string",
+                "description": "Indicates whether the node is functioning or not.",
+                "enum": [
+                  "Up",
+                  "Down"
+                ]
+              },
+              "state": {
+                "type": "string",
+                "description": "The state of the node in relation to the cluster.",
+                "enum": [
+                  "Normal",
+                  "Leaving",
+                  "Joining",
+                  "Moving",
+                  "Stopped"
+                ]
+              },
+              "address": {
+                "type": "string",
+                "description": "The node's URL."
+              },
+              "load": {
+                "type": "string",
+                "description": "The amount of file system data in the data directory (e.g., 47.66 KB), excluding all content in the snapshots subdirectories. Because all SSTable data files are included, any data that is not cleaned up (such as TTL-expired cell or tombstoned data) is counted."
+              },
+              "tokens": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The number of tokens set for the node."
+              },
+              "owns": {
+                "type": "number",
+                "format": "double",
+                "description": "The percentage of the data owned by the node per datacenter times the replication factor (e.g., 33.3, or null if the data is not available). For example, a node can own 33% of the ring, but shows 100% if the replication factor is 3. For non-system keyspaces, the endpoint percentage ownership information is shown."
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "ListBackups": {
+      "description": "List of restorable backups for a Cassandra cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of backups.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
```

Comment:
>id [](start = 15, length = 2)

since this is returned for a list backups call - it must include name, id and type, timestamp must be in a custom property bag

Reply:
Turned it into a proxy resource.